assets/js/blocks/cart-checkout/mini-cart/block.tsx

```diff
@@ -29,13 +29,13 @@ import QuantityBadge from './quantity-badge';
 import MiniCartContentsBlock from '../mini-cart-contents/block';
 import './style.scss';
 
-interface Props {
+type Props = {
```

Comment:
Just curious, what is the reason for the change?

Reply:
Not really. I am used to working with `type` and while experimenting I made this change. Restored it